hyrax/app/inputs/nested_attributes_input.rb

```diff
@@ -118,15 +118,21 @@ def id_for(attribute_name, index, field, parent=@builder.object_name)
     end
 
     def class_for(attribute_name, field, model_class=@builder.object.class.model_class)
```

Comment:
to "require-if" or not to "require-fi" that is the only question that class_for now answers